src/openforms/formio/components/vanilla.py

```diff
@@ -348,9 +429,16 @@ def build_serializer_field(
 
 
 @register("signature")
-class Signature(BasePlugin):
+class Signature(BasePlugin[SignatureComponent]):
     formatter = SignatureFormatter
 
+    def build_serializer_field(
+        self, component: SignatureComponent
+    ) -> serializers.CharField:
+        validate = component.get("validate", {})
+        required = validate.get("required", False)
+        return serializers.CharField(required=required, allow_null=not required)
```

Comment:
I think `allow_blank` is preferred? https://www.django-rest-framework.org/api-guide/fields/#charfield